app/src/main/java/org/wikipedia/page/PageToolbarHideHandler.java

```diff
@@ -1,95 +1,20 @@
 package org.wikipedia.page;
 
-import android.animation.ArgbEvaluator;
-import android.graphics.Color;
-import android.graphics.PorterDuff;
-import android.graphics.drawable.Drawable;
 import android.view.Gravity;
 import android.view.View;
-import android.view.ViewGroup;
-import android.widget.ImageView;
 
-import androidx.annotation.ColorInt;
 import androidx.annotation.NonNull;
-import androidx.appcompat.widget.Toolbar;
 
-import org.wikipedia.R;
-import org.wikipedia.util.DimenUtil;
 import org.wikipedia.views.TabCountsView;
 
-import static org.wikipedia.util.ResourceUtil.getThemedColor;
-
 public class PageToolbarHideHandler extends ViewHideHandler {
-    private static final int FULL_OPACITY = 255;
-
-    private ArgbEvaluator argbEvaluator = new ArgbEvaluator();
-    private boolean fadeEnabled;
-    private Boolean prevToolbarLight;
-    @NonNull private PageFragment pageFragment;
-    @NonNull private Toolbar toolbar;
-    @NonNull private Drawable toolbarBackground;
-
-    @ColorInt private int themedIconColor;
 
-    private int toolbarHeight;
-
-    PageToolbarHideHandler(@NonNull PageFragment pageFragment, @NonNull View hideableView,
-                                  @NonNull Toolbar toolbar, @NonNull TabCountsView tabsButton) {
+    PageToolbarHideHandler(@NonNull View hideableView, @NonNull TabCountsView tabsButton) {
         super(hideableView, null, Gravity.TOP);
-        this.pageFragment = pageFragment;
-        this.toolbar = toolbar;
-        this.toolbarBackground = hideableView.getBackground().mutate();
-        themedIconColor = getThemedColor(toolbar.getContext(), R.attr.toolbar_icon_color);
-        toolbarHeight = DimenUtil.getToolbarHeightPx(pageFragment.requireContext());
         tabsButton.updateTabCount();
```

Comment:
Since this class is nearly empty, should we move this `tabsButton.updateTabCount();` to `PageActivity` and remove this class?

Reply:
yep!